Processing/src/cz/fidentis/processing/featurePoints/LandmarkLocalization.java

```diff
@@ -0,0 +1,302 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package cz.fidentis.processing.featurePoints;
+
+
+import Jama.Matrix;
+import cz.fidentis.comparison.icp.ICPTransformation;
+import cz.fidentis.comparison.icp.KdTreeIndexed;
+import cz.fidentis.comparison.procrustes.ProcrustesAnalysis;
+import cz.fidentis.featurepoints.FacialPoint;
+import cz.fidentis.featurepoints.FeaturePointsUniverse;
+import cz.fidentis.featurepoints.FpModel;
+import cz.fidentis.featurepoints.pdm.TrainingModel;
+import cz.fidentis.featurepoints.pdm.CSVparsers;
+import cz.fidentis.featurepoints.pdm.AreasSearch;
+import cz.fidentis.model.Model;
+import cz.fidentis.model.corner_table.CornerTable;
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+import javax.vecmath.Vector3f;
+import jv.vecmath.PdVector;
+
+/**
+ *
+ * @author Rasto1
+ */
+public class LandmarkLocalization {
+    
+    private static final int[] FPTYPES = {1,2,3,4,5,6,7,8,9,10,11,12,13,14};
+    
+    /**
+     * Landmark localization method, compute landmarks
+     * @param model model for localization
+     * @param trainingShapesFiles .csv shapes for training model
+     * @return List of 14 landmarks
+     */
+    public List<FacialPoint> makeArea(Model model, File[] trainingShapesFiles) {
```

Comment:
lepsi nazov metody, make area mi nijak neevokuje co sa v metode deje (co to ma spolocne s trenovanim modelu, alebo vyhladavanim landmarkov?)